moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -54,7 +54,7 @@ namespace core
  * valid paths from paths with large joint space jumps. */
 static const std::size_t MIN_STEPS_FOR_JUMP_THRESH = 10;
 
-const std::string LOGNAME = "robot_state";
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit.robot_state");
```

Comment:
This won't compile, refer to https://github.com/ros-planning/moveit2/pull/8#discussion_r266211476

Reply:
Ok, we can deal with addressing the ``static`` aspect in the future.

Reply:
```suggestion
const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit.robot_state");
```